### DISCUSSION TITLE
Two axes poll

### DISCUSSION THREAD

**Discussion by LeonLeCame** [Zuehlke/poinz#335] [Ideas]
Hello,
Would it be possible to allow chosing two metrics for a single story ? Typically, we would like to determine the business value at the same time as the effort. Then I would need to select two cards (scale can be identical).
Thanks

**Comment by xeronimus** [comment]
Hello @LeonLeCame ,

thanks for the suggestion. Until now, I never thought about that...

To be honest, my first thought was, I'm not sure whether the added complexity is worth it. We always try to keep PoinZ as simple and easy to use as possible.
